ui/src/components/VolumeListTable.js

```diff
@@ -490,32 +153,19 @@ const VolumeListTable = (props) => {
       },
     ];
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [volumeListData, theme, isNodeColumn]);
-  const nodeCol = {
-    Header: 'Node',
-    accessor: 'node',
-    cellStyle: {
-      textAlign: 'left',
-      flex: 1,
-      paddingLeft: '0.857rem',
-      minWidth: '3.214rem',
-    },
-  };
-  if (isNodeColumn) {
-    columns.splice(2, 0, nodeCol);
-  }
+  }, [volumeListData, theme]);
 
   // handle the row selection by updating the URL
   const onClickRow = (row) => {
     const query = new URLSearchParams(location.search);
-    const isAddNodeFilter = query.has('node');
+    const isAddNodeFilter = query.has('search');
     const isTabSelected =
       location.pathname.endsWith('/alerts') ||
       location.pathname.endsWith('/metrics') ||
       location.pathname.endsWith('/details');
 
-    if (isAddNodeFilter || !isNodeColumn) {
-      history.push(`/volumes/${row.values.name}/overview?node=${nodeName}`);
+    if (isAddNodeFilter) {
+      history.push(`/volumes/${row.values.name}/overview?search=${nodeName}`);
```

Comment:
```suggestion
      history.push(`/volumes/${row.values.name}/overview?${query.toString()}`);
```
and above condition can be dropped